lib/alerts_viewer_web/live/bus_live.ex

```diff
@@ -121,13 +127,32 @@ defmodule AlertsViewerWeb.BusLive do
     """
   end
 
+  def severity_to_minutes(severity) when severity < 3, do: "<10"
+  def severity_to_minutes(severity) when severity >= 9, do: "60+"
+
+  def severity_to_minutes(severity) do
```

Comment:
thought: What you have here works fine, but another approach might be to list each case as an individual function definition with exact pattern matching, e.g.:

    def severity_to_minutes(3), do: "10"

Reply:
👍🏻